Doc/library/sqlite3.rst

```diff
@@ -525,18 +584,28 @@ Connection objects
    .. method:: commit()
 
       Commit any pending transaction to the database.
-      If there is no open transaction, this method is a no-op.
+      If :attr:`autocommit` is :const:`True`,
+      or there is no open transaction,
+      this method is a no-op.
```

Comment:
This almost looks like poetry, the lines are so short!

Reply:
Reflowing :)